recipes/scc/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.tools.files import get
+from conans import tools, CMake
+from conan.errors import ConanInvalidConfiguration
+import functools
+
+required_conan_version = ">=1.43.0"
+
+class SystemcComponentsConan(ConanFile):
+    name = "scc"
+    description = """A light weight productivity library for SystemC and TLM 2.0"""
+    homepage = "https://minres.github.io/SystemC-Components"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("systemc", "modeling", "tlm", "scc")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "sc_with_phases_callbacks": [True, False],
+        "sc_with_phases_callback_tracing": [True, False]
```

Comment:
We usually dont have the project name for options, when you pass them to conan you need to pass the name so it looks odd `-o scc:sc_with_callbacks` I understand it matches upstream so just a suggestions

 https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#recommended-feature-options-names

Maybe something like?

```suggestion
        "enable_phases_callbacks": [True, False],
        "enable_phases_callback_tracing": [True, False]
```